pkg/codegen/internal/test/testdata/simple-enum-schema/dotnet/Enums.cs

```diff
@@ -0,0 +1,47 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System.Runtime.Serialization;
+
+namespace Pulumi.PlantProvider
+{
+    /// <summary>
+    /// plant container colors
+    /// </summary>
+    public readonly struct ContainerColor : IEquatable<ContainerColor>
+    {
+        private readonly string _value;
+
+        private ContainerColor(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        public static ContainerColor Red { get; } = new ContainerColor("red");
+        public static ContainerColor Blue { get; } = new ContainerColor("blue");
+        public static ContainerColor Yellow { get; } = new ContainerColor("yellow");
+
+        public static bool operator ==(ContainerColor left, ContainerColor right) => left.Equals(right);
+        public static bool operator !=(ContainerColor left, ContainerColor right) => !left.Equals(right);
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Note: `EditorBrowsableAttribute` and `EditorBrowsableState` are in the `System.ComponentModel` namespace, so we'll need to emit a `using System.ComponentModel;` at the top of the file (or, alternatively, fully qualify in the emitted code).

Reply:
Heh, yeah I realized I had some imports missing once I started working through the SDK changes.